dle/.env.example

```diff
@@ -77,18 +77,29 @@ ELASTICSEARCH_USER=elastic
 
 # Elastic ingest
 # ------------------------------------------------------------------------------
-# In the entrypoint, whether to load Django data to Elasticsearch; whether to create an ES NLP pipeline with the BERT model (script doesn't exist yet);
-# and/or whether to load vectors from an external source (don't exist yet);
-# LOAD_DJANGO_TO_ES=True
-# LOAD_VECTORS=False
-# If ES_AUTO_SYNC is True, elasticsearch-django will update the index as data is changed (including on initial ingest)
+# In the entrypoint, whether to load data from vector JSON files into Django
+# Doesn't work well because of complex composite keys / nested structure
+# LOAD_FDA_VECTORS=False
+# LOAD_TGA_VECTORS=False
+# LOAD_EMA_VECTORS=False
```

Comment:
and HC? Do we still need to keep these load vectors if loading fixtures works?

Reply:
Loading fixtures sort of works but can also cause containers to crash due to RAM limits ... I think that a PSQL dump is going to be best